Readme.rst

```diff
@@ -45,3 +45,27 @@ This repository is a collection of packages for the Flow framework (learn more o
 The repository is used for development and all pull requests should go into it.
 
 If you want to use the Flow framework, please have a look at the documentation: https://flowframework.readthedocs.org/en/latest/
+
+Contributing
+============
+
+If you want to contribute to Flow framework and want to set up a development environment, then follow these steps:
+
+``composer create-project neos/flow-development-distribution flow-development dev-master --keep-vcs``
+
+Note the **-distribution** package you create a poject from, instead of just checking out this repository.
+
+The code for the framework is then found inside ``Packages/Framework``, which is itself the git repository for this flow-development-collection (due to the ``--keep-vcs`` option above), that you need to apply changes to and create pull requests for.
```

Comment:
This sentence is complicated to read. Suggestion:
The code of the framework can then be found inside ``Packages/Framework``, which itself is the flow-development-collection Git repository (due to the ``--keep-vcs`` option above). You commit changes and create pull requests from this repository.